src/atoms/player-button/index.jsx

```diff
@@ -1,22 +1,29 @@
-import React, { useMemo, useState } from "react";
```

Comment:
import "React" from React is no longer necessary with current react edition. Try it out. Should work.